crates/wash-lib/src/parser/mod.rs

```diff
@@ -403,7 +403,10 @@ pub fn get_config(opt_path: Option<PathBuf>, use_env: Option<bool>) -> Result<Pr
             path,
         )
     } else {
-        bail!("no wasmcloud.toml file found in {}", path.display());
+        bail!(
+            "unrecognized path [{}] (neither directory nor file)",
+            path.display()
+        );
```

Comment:
So I'm not sure... because we're basically not even *trying* to find it at that point, with how the code is written. We don't give up because we couldn't find it, we give up because the path is not a dir or a file, really....

I'm not against adding a prefix of `failed to find wasmcloud.toml:` though

Reply:
made a slight change to the messages:

https://github.com/wasmCloud/wasmCloud/pull/1870/commits/0c7dd162e446bd525ce093e50d3df63a39260e29